src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/content_key_policy.py

```diff
@@ -105,43 +105,87 @@ def remove_content_key_policy_option(client, resource_group_name, account_name,
+            elif token_key_type == 'X509':
+                policy_option.restriction.primary_verification_key = _x509_token_key_factory(token_key)
+
+        if alt_token_key is not None and alt_token_key_type is not None:
+            if alt_token_key_type == 'Symmetric':
+                policy_option.restriction.alternate_verification_keys.append(_symmetric_token_key_factory(alt_token_key))
+            elif alt_token_key_type == 'RSA':
+                policy_option.restriction.alternate_verification_keys.append(_rsa_token_key_factory(alt_token_key))
+            elif alt_token_key_type == 'X509':
+                policy_option.restriction.alternate_verification_keys.append(_x509_token_key_factory(alt_token_key))
+
+        if token_claims is not None:
+            policy_option.restriction.token_claims = []
+            for key in token_claims:
+                claim = ContentKeyPolicyTokenClaim(claim_type=key,
+                                                   claim_value=token_claims[key])
+                policy_option.restriction.token_claims.append(claim)
+
+        if token_type is not None:
+            policy_option.restriction.restriction_token_type = token_type
+
+        if open_id_connect_discovery_document is not None:
+            policy_option.restriction.open_id_connect_discovery_document = open_id_connect_discovery_document
+
+    if isinstance(policy_option.configuration, ContentKeyPolicyWidevineConfiguration):
+        if widevine_template is not None:
+            policy_option.configuration = ContentKeyPolicyWidevineConfiguration(widevine_template=widevine_template)
+    elif isinstance(policy_option.configuration, ContentKeyPolicyFairPlayConfiguration):
+        if ask is not None:
+            policy_option.configuration.ask = bytearray(ask, 'utf-8')
+
+        if fair_play_pfx_password is not None:
+            policy_option.configuration.fair_play_pfx_password = fair_play_pfx_password
+
+        if fair_play_pfx is not None:
+            policy_option.configuration.fair_play_pfx = _b64_to_str(_read_binary(fair_play_pfx)).decode('ascii')
+
+        if rental_and_lease_key_type is not None:
+            policy_option.configuration.rental_and_lease_key_type = rental_and_lease_key_type
+
+        if rental_duration is not None:
+            policy_option.configuration.rental_duration = rental_duration
+    elif isinstance(policy_option.configuration, ContentKeyPolicyPlayReadyConfiguration):
+        if play_ready_template is not None and _valid_playready_configuration(play_ready_template):
+            _play_ready_configuration_factory(json.loads(play_ready_template))
 
+    return client.update(resource_group_name, account_name,
+                         content_key_policy_name, policy.options, policy.description)
+
+
+def update_content_key_policy_setter(client, resource_group_name, account_name, content_key_policy_name,
```

Comment:
As we are not using setter now, we can remove it... just set setter name in commands.py with 'update' value to override the default setter value (create_or_update)

Reply:
As previously discussed, it won't work if we don't use a custom setter.

Reply:
Got it